packages/desktop-gui/cypress/integration/settings_spec.js

```diff
@@ -181,6 +182,55 @@ describe('Settings', () => {
           expect(this.ipc.externalOpen).to.be.calledWith('https://on.cypress.io/guides/configuration')
         })
       })
+
+      it('displays env settings', () => {
+        cy.get('@config').then(({ resolved }) => {
+          const getEnvKeys = flow([
+            get('env'),
+            toPairs,
+            map(([key]) => key),
+            sortBy(get('')),
+          ])
+
+          cy.contains('.line', 'env').contains(flow([getEnvKeys, join(', ')])(resolved))
+          cy.contains('.line', 'env').click()
```

Comment:
these lines should go below the `const ... =` assignments because the execute async.

It's generally better to group the sync/async code together